packages/api/src/external/carequality/document/shared.ts

```diff
@@ -87,3 +92,60 @@ const generateCQFHIRContent = (
 
   return [cqFHIRContent];
 };
+
+function generateCQFHIRContained(authorInstitution: string | undefined): Organization | undefined {
+  if (!authorInstitution) return;
+
+  const org = splitNameAndOid(authorInstitution);
+  if (!org) return generateOrganization(authorInstitution);
+  const organization = generateOrganization(org.name, org.oid);
```

Comment:
if both `org.name` and `org.oid` are undefined, we probably don't want to generate the org, and use `authorInstitution` instead, right?

Reply:
I think if we can't parse the string into name and oid, then we just consider it a name and generateOrganization.

The usual format of org names is `CompanyName^^^^^^1.22.333.4444`